editor/src/clj/editor/settings_core.clj

```diff
@@ -188,9 +209,24 @@
 (defn- category->str [category settings]
   (s/join "\n" (cons (str "[" category "]") (map setting->str settings))))
 
+(defn split-multi-line-setting [category key settings]
```

Comment:
This is where we split a comma separated settings value into multi-line as part of converting this into a string when saving to disk.